src/biome/text/configuration.py

```diff
@@ -260,7 +261,6 @@ def __init__(
         self.optimizer = optimizer
         self.validation_metric = validation_metric
         self.patience = patience
-        self.shuffle = shuffle
         self.num_epochs = num_epochs
         self.cuda_device = cuda_device
         self.grad_norm = grad_norm
```

Comment:
As I understand, iterator params, except batch_size will have no effect for now. I am ok to leave them for now, and discuss what to do about lazy vs. normal dataset readers, we just need to remember to clean up.

Reply:
For now, no bucketing is enabled during training phase. 

We can discuss this behaviour in another PR since there is no easy solution